DoctorWho/DoctorWho.Db/Repositories/EpisodeRepository.cs

```diff
@@ -0,0 +1,39 @@
+﻿
+namespace DoctorWho.Db.Repositories
+{
+    public class EpisodeRepository
+    {
+        public static void CreateEpisode(int seriesNumber, int episodeNumber, string episodeType, string title, int authorId, int doctorId, DateTime date)
```

Comment:
if you have 20 columns, then you will need to pass them one by one and this will get bigger. Consider using a new single model to pass instead. Same comment for other function that has many params.